source/kraken/apply_disruption.cpp

```diff
@@ -220,18 +220,59 @@ struct add_impacts_visitor : public apply_impacts_visitor {
             for (const auto& stu: impact->aux_info.stop_times) {
                 stoptimes.push_back(stu.stop_time);
             }
+
+            // Create new VJ
             auto* vj = mvj->create_discrete_vj(new_vj_uri,
                 type::RTLevel::RealTime,
                 canceled_vp,
                 r,
                 std::move(stoptimes),
                 pt_data);
+            LOG4CPLUS_TRACE(log, "New vj has been created " << vj->uri);
+
+            // Add company
             if (!impact->company_id.empty()) {
+                LOG4CPLUS_TRACE(log, "kikou " << impact->company_id);
```

Comment:
Not sure you wanna keep this trace :smile: 
If you want to keep it, Alex's pseudo is ki`nn`ou :wink: 
(don't know about the others also, but they may help)

Reply:
hahaha my famous **kikou message** :speak_no_evil: